### PR TITLE
PolicyInsights: Add policy version columns

### PR DESCRIPTION
**Pull request by pilor**
Added the read-only policy entity version columns to the OData query responses in the Microsoft.PolicyInsights/policyStates API.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json [hunk 1]
```diff
@@ -1228,6 +1228,21 @@
...
+          "type": "string",
+          "readOnly": true
+        },
+        "policySetDefinitionVersion": {
+          "description": "Evaluated policy set definition version.",
+          "type": "string",
+          "readOnly": true
+        },
+        "policyAssignmentVersion": {
+          "description": "Evaluated policy definition version.",
```
Comment (sandipsh) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json]: >definition [](start = 43, length = 10)

assignment

**Review by sandipsh — APPROVED** [review]
:shipit:

**Review by lirenhe — APPROVED** [review]

**Comment by pilor** [comment]
@ravbhatnagar can you review this when you get a chance.  Its very small

**Comment by pilor** [comment]
@anthony-c-martin Gaurav didn't get to this last week.  Its very small, can you take a look?

**Comment by anthony-c-martin** [comment]
@pilor LGTM - have signed off

**Comment by pilor** [comment]
@lirenhe ready for merge

**Pull request merged**